src/start_testbuilds.py

```diff
@@ -147,7 +156,7 @@ def main():
             if "graviton" in images_str and test_type == "sanity":
                 pr_test_job += "-graviton"
             if is_test_job_enabled(test_type) and is_test_job_implemented_for_framework(images_str, test_type):
-                run_test_job(commit, pr_test_job, images_str)
+                run_test_job(commit, pr_test_job, args.buildspec, images_str)
```

Comment:
The buildspec file needs to be a compulsory input to the `start_testbuilds.py` script, and is therefore passed in through the same method by which we pass it to the `main.py` script.

The arg being passed in is already obtained from the environment variable. Do you recommend defaulting to the environment variable inside the python code in both the `main.py` and the `start_testbuilds.py` cases? Because that change is okay with me.

Reply:
Defaulting it to a value makes sense to me. It is just my common practice to check any parameter passed to a function before using it within the function.

Reply:
Made the change to default to the value obtained from the environment variable `FRAMEWORK_BUILDSPEC_FILE`.